CMS/enums.py

```diff
@@ -8,6 +8,29 @@ class LanguagesFull:
     WELSH = "welsh"
 
 
+class Countries:
+    ENGLAND = 'XF'
+    WALES = 'XI'
+    SCOTLAND = 'XH'
+    IRELAND = 'XG'
+
+
+class APRValues:
+    MET = 'Meets requirements'
+    ACTION_PLAN = 'Meets requirements with an action plan'
+    PENDING = 'Pending'
+    NOT_MET = 'Does not meet requirements'
+
+
+class TEFValues:
+    GOLD = 'Gold'
+    SILVER = 'Silver'
+    BRONZE = 'Bronze'
+    PROVISIONAL = 'Provisional'
+    NOT_PARTICIPATE = 'Did not participate'
+    WITHDRAWN = 'Withdrawn'
```

Comment:
This needs to be confirmed with Catherine as to what it is called in HESA xml; just a reminder

Reply:
Confirmed Withdrawn shouldn't be in the original dataset - but there is no need to remove it being handled for now